main.go

```diff
@@ -396,7 +397,12 @@ func listFavorites(cCtx *cli.Context) error {
 	// print it out
 	if cCtx.Bool("verbose") {
 		for _, f := range fMap {
-			fmt.Printf(" %v:\n   account number: %v\n   cloud access role: %v\n", f.Name, f.Account, f.CAR)
+			// Check if AccessType is defined; if not, set a default value of "web" in the verbose output
+			accessType := f.AccessType
+			if accessType == "" {
+				accessType = "web (Default)"
```

Comment:
I know you didn't make web the default, but I feel pretty strongly that stak should be default.  Prior to adding web support, stak was the only option.  Existing config files won't have an access_type, and making web default will change the behavior for existing config files.  Plus this is a CLI tool, where IMO the more common use case is stak over console acces.

But again, I know you didn't make this change so I don't think we need to change in this PR.

Reply:
Yeah, I'll do the PR now.

Reply:
It's a simple fix to make staks the default, let's go ahead and do that. Should just be here and in the favorites func. Just reversing the flow on line 341.